src/makeCalculator.js

```diff
@@ -4,7 +4,49 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  let resultValue = 0;
+
+  return {
+    add(param) {
+      resultValue += param;
+
+      return this;
+    },
+
+    subtract(param) {
+      resultValue -= param;
+
+      return this;
+    },
+
+    divide(param) {
+      resultValue /= param;
+
+      return this;
+    },
+
+    multiply(param) {
+      resultValue *= param;
+
+      return this;
+    },
+
+    reset() {
+      resultValue = 0;
+
+      return this;
+    },
+
+    operate(callback, param) {
+      callback(param);
+
+      return this;
```

Comment:
The `operate` method should update the `resultValue` with the result of the callback execution. Currently, the callback is called with the parameter, but the `resultValue` is not being updated. To fix this, you should assign the result of the callback to `resultValue`. For example: `resultValue = callback.call(this, resultValue, param);`